src/app/page-courses/page-courses.component.spec.ts

```diff
@@ -19,6 +37,10 @@ describe('PageCoursesComponent', () => {
 		fixture.detectChanges();
 	});
 
+	it('listCourses', () => {
+		expect(component.listCourses).toEqual(arr)
```

Comment:
This test needs to be rewriten after implementing a service for cources receiving

Reply:
Of course